taskboot/docker.py

```diff
@@ -442,7 +462,7 @@ def push_archive(self, path, custom_tag=None):
                 "copy",
                 "--authfile",
                 self.auth_file,
-                "oci-archive:{}".format(path),
+                "docker-archive:{}".format(path),
```

Comment:
Is this OK for docker images built by any tool?

Reply:
The errors I get for each case:

```
/taskboot # skopeo inspect oci-archive:docker.tar
FATA[0000] Error parsing image name "oci-archive:docker.tar": open /var/tmp/oci3618682673/index.json: no such file or directory
/taskboot # skopeo inspect docker-archive:docker.tar
FATA[0000] Error retrieving manifest for image: Unexpected tar manifest.json: expected 1 item, got 3
/taskboot # skopeo inspect oci-archive:podman.tar
FATA[0000] Error parsing image name "oci-archive:podman.tar": open /var/tmp/oci3931677334/index.json: no such file or directory
```

Reply:
My mistake. Specifying the tag with `alpine:latest` fixes this problem:

| command | oci-archive: | docker-archive: |
| -- | -- | -- |
| docker | ❌ | ✔️ |
| img | ✔️ | ✔️ |
| podman | ❌ | ✔️ |

Reply:
Sounds good.
Alpine's latest tag currently corresponds to 3.17.